gips/0001-gip-process.md

```diff
@@ -35,80 +35,138 @@ The high-level process is captured in the diagram below.
+
 ## Writing a proposal
 
-There are three primary types of documents that are tracked in this repo. These are tracked in separate folders with their own numbering:
+There are two primary types of documents that are tracked in this repo. These are tracked in separate folders with their own numbering:
 
 - **Graph Improvement Proposals (GIPs).** Describes some improvement to the protocol rules, interfaces, software functionality, protocol charters, protocol parameters, processes, etc.
-- **Graph Requests for Proposals (GRPs)** Formalizes a problem or opportunity for the protocol to address without putting forth a specific solution.
 - **Graph Requests for Comment (GRCs)** Specifies an application-level community standard. For example, this could be a common way of designing subgraphs for specific use cases or to support better composability between applications.
 
 Depending on the type of proposal, they will move through the following stages, as indicated by the diagram up above:
 
-1. **Strawperson.** A proposal at this stage describes an idea or problem that the proposal aims to solve but doesn't expand too much on the specifics of a solution. In terms of the document sections outlined below, a proposal at this stage would have a well-defined "Abstract" and "Motivation" and optionally a rough draft of a "High-Level Description.".
-2. **Proposal.** Describes how a proposal works at a high level, without getting too in the weeds with specific APIs and technical implementation details. Sections relevant to this stage might include a more well-developed "High-Level Description", "Prior Art," "Dependencies," "Backwards Compatibility", etc.
-3. **Draft.** This is a detailed technical specifical of a proposal. It includes specific API changes, implementation details, architecture, etc. Relevant document sections might include "Detailed Specification", "Risks and Security Considerations," "Validation," "Rationale and Alternatives".
-4. **Candidate.** A proposal at this stage has a reference implemenetation that has been linked in the document and can be used to validate the idea described in the protocol.
-5. **Accepted | Deferred | Withdrawn | Living.** These are resolution stages for a proposal:
+1. **Draft.** This stage encompasses all proposals that are not ready for deployment. The proposal might include just Abstract and Motivation, or have a full-blown Detailed Specification. It can evolve through community feedback when in this phase, and the key requirement to move to "Candidate" is to have an implementation that is ready for deployment.
+2. **Candidate.** A proposal at this stage has a reference implementation that has been linked in the document and can be used to validate the idea described in the protocol. The implementation must be ready for deployment, having completed any relevant audits and test plans.
```

Comment:
I have also started using 'Reserved', for GIP numbers that are reserved for future use but might not have any details associated yet. This is an optional stage before 'Draft'. If necessary or simpler they could have 'Draft' status and state they are reserved in the text.

Reply:
I like the idea of adding an "Implemented" stage. 
And for simplicity @RembrandtK I would suggest just using Draft, rather than adding an additional stage for Reserved.

For clarity it would be good if we outlined the "happy path" first, i.e. Draft-Candidate-Approved-Implemented, and then separately mention the other possible states (Deferred/Withdrawn), so that readers can get a general idea of the usual flow and then learn about the "exceptions". Wdyt?

Reply:
Agreed, we should present Deferred and Withdrawn after the natural progression of stages.  I also agree having one stage for draft reduces complexity without losing effectiveness.